### DISCUSSION TITLE
Apkm file

### DISCUSSION THREAD

**Discussion by Otrovert** [ReVanced/revanced-discussions#1060] [Question]
There is a problem. Some applications are now published in 'xapk' or 'apkm' format, how do we patch these applications with Revance Manager? I have applications that need to be updated (for example Twitter app), there must be a solution. If there is a solution, please tell me.

**Comment by oSumAtrIX** [comment]
ReVanced Manager requires an APK file

**Reply by Otrovert** [reply]
OK, this is already known, for example, the Twitter app is no longer published in apk format, is there anything we can do about it?

**Reply by oSumAtrIX** [reply]
This is outside of the scope of ReVanced. You will need an APK file. Google instantly returned results to your question.

**Reply by Otrovert** [reply]
Some apps are now released as 'apk bundles' and cannot be patched on non-root devices, for example the twitter app. Google search could not fix this problem, you can try and see.

**Comment by YvleUml** [comment]
Antisplit m, it is present on github.

**Comment by StickyChannel92** [comment]
This issue is still prevalent today. I have an apkm and Revanced does not accept it. Neither can it patch an already-installed version of the unmodified app from Google Play.
